openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AddAndRemoveTierAndOwner.spec.js

```diff
@@ -41,28 +41,55 @@ const ENTITIES = {
 const glossary = 'GlossaryOwnerTest';
 const glossaryTerm = 'GlossaryTermOwnerTest';
 
-const OWNER = 'Aaron Singh';
+const OWNER = 'Amber Green';
 const TIER = 'Tier1';
 
-const addRemoveOwner = () => {
+const addRemoveOwner = (isGlossaryPage) => {
   cy.get('[data-testid="edit-owner"]').click();
 
   cy.get('.ant-tabs [id*=tab-users]').click();
   verifyResponseStatusCode('@getUsers', 200);
+
+  interceptURL(
+    'GET',
+    `api/v1/search/query?q=*${encodeURI(OWNER)}*`,
+    'searchOwner'
+  );
+
+  cy.get('[data-testid="owner-select-users-search-bar"]').type(OWNER);
+
+  verifyResponseStatusCode('@searchOwner', 200);
+
   cy.get(`.ant-popover [title="${OWNER}"]`).click();
   verifyResponseStatusCode('@patchOwner', 200);
-  cy.get('[data-testid="owner-link"]').should('contain', OWNER);
+  if (isGlossaryPage) {
+    cy.get('[data-testid="glossary-owner-name"]')
+      .should('be.visible')
+      .should('contain', OWNER);
+  } else {
+    cy.get('[data-testid="owner-link"]').should('contain', OWNER);
+  }
   cy.get('[data-testid="edit-owner"]').click();
 
   cy.get('[data-testid="remove-owner"]').click();
   verifyResponseStatusCode('@patchOwner', 200);
-  cy.get('[data-testid="owner-link"]').should('contain', 'No Owner');
+  if (isGlossaryPage) {
+    cy.get('[data-testid="glossary-owner-name"] > [data-testid="Add"]').should(
+      'be.visible'
+    );
+  } else {
+    cy.get('[data-testid="owner-link"]').should('contain', 'No Owner');
+  }
 };
 
 const addRemoveTier = () => {
   cy.get('[data-testid="edit-tier"]').click();
-  cy.get('[data-testid="card-list"]').first().should('be.visible').as('tier1');
-  cy.get('@tier1').find('[data-testid="radio-btn"]').click();
+  cy.get('[data-testid="radio-btn-Tier1"]').check();
+
+  cy.wait(2000);
```

Comment:
why do we need manual wait? we should wait on API call

Reply:
While running on the CI, there was flakiness because of this, the radio button was not being checked quickly. So added wait which resolved that

Reply:
@aniketkatkar97 Let's not use manual wait